project/src/components/comment-form/comment-form.tsx

```diff
@@ -0,0 +1,141 @@
+import { FormEvent, useState } from 'react';
+
+function CommentForm():JSX.Element {
+  const [formData, setFormData] = useState({
+    rating: '',
+    comment: '',
+  });
+
+  const ratingChangeHandler = (evt:FormEvent<HTMLInputElement>) => {
+    const value = evt.target;
+    setFormData({...formData, rating: String(value)});
```

Comment:
```suggestion
    const value = evt.target.value;
    setFormData({...formData, rating: String(value)});
```
это вообще работает? таргет это же будет просто дом-нода?